rpm-new/build_repos.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/bash
+
+set -euo pipefail
+
+assets_dir=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )/assets
+output_dir="$(realpath "$1")"
+
+# Complex repository - use all metadata features
+
+mkdir -p "$output_dir"
+
+while IFS= read -r filename
+do
+  cp --no-preserve=mode --reflink=auto "$assets_dir/packages/$filename" "$output_dir/$filename" ;
+done < "$assets_dir/complex_repo_pkglist.txt"
```

Comment:
We should make this more generic if / when we re-use it but for now this is fine.